addons/core/XEH_postInit.sqf

```diff
@@ -2,6 +2,9 @@
 
 if (!isMultiplayer && !is3DENMultiplayer) exitWith {}; //Don't do anything in Singleplayer
 
+if(isMultiplayer && isServer ) then {
```

Comment:
isMultiplayer check doesn't make sense here.
Also whitespace after the `if` and no whitespace before the closing `)`